pkgs/applications/science/math/mxnet/default.nix

```diff
@@ -2,6 +2,7 @@
 , opencv3, gtest, blas, gomp, llvmPackages, perl
 , cudaSupport ? config.cudaSupport or false, cudatoolkit, nvidia_x11
```

Comment:
Hmm, maybe I should get rid of this `nvidia_x11` while I'm at it? But this would demand some testing

Reply:
If you're up for it, but it's also prob not a blocker for this PR

Reply:
I guess it's safer to keep focus